docs/integrations/code-conventions.md

```diff
@@ -600,6 +600,42 @@ You may also use ```headerTransform``` to convert the existing keys into formatt
  
  <img width="758" src="../doc_imgs/integrations/50575199-fd5d0a00-0e01-11e9-9d54-944eb7c6f287.png"></img>
 
+### IndicatorsTimeline
+The `IndicatorTimeline` is an optional object (available from Server version 5.5.0 and up) . It is only applicable for commands that operate on indicators. It is a dictionary (or list of dictionaries) of the following format:
+When `IndicatorTimeline` data is returned in an entry, the timeline section of the indicator whose value was noted in the timeline data will be updated (and is viewable in the indicator's view page in Cortex XSOAR as can be seen in the attached image).
+
+<img width="758" src="../doc_imgs/integrations/timeline_section.png"></img>
+
+**What value should be used for the `'Category'` field of a `timeline` data object?**  
+Any Cortex XSOAR integration command that returns `timeline` data may include the `'Category'` value.
+If not given, When returning `timeline` data from a Cortex XSOAR integration or automation, the value will be `'Integration Update'` or `'Automation Update'` accordingly.
```

Comment:
You didn't answer the question. You didn't provided valid Category examples.
@cloughrm which values are expected to be filled in timeline `Category`?

Reply:
Aside from integration/automation update, I'm not familiar with any. let's wait for an example from ryan.